yarn/chapter.md

```diff
@@ -374,7 +374,8 @@ When fixes for these bugs are merged into master, they are added to the version
 After that, the changes are added to npm, and the CI builds and deploys the new version.
 
 #### Feature Flags
-<div id="feature-flags" />
+<div id="feature-flags"></div>
```

Comment:
I would vouch for merging. That seems more future-safe, if we want to later change headers if we need to.